plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2418,6 +2418,12 @@ public boolean supportsReportingWrittenBytes(ConnectorSession session, SchemaTab
         return true;
     }
 
+    @Override
+    public boolean supportsMultipleWritersPerPartition(ConnectorSession session, SchemaTableName schemaTableName)
+    {
+        return true;
```

Comment:
cc @findepi

Reply:
👍  Iceberg has no constraint on number of files per partition, regardless of partitioning transform (function).